.github/workflows/release-process.yml

```diff
@@ -40,10 +40,18 @@ jobs:
           cd cli
           ./hack/build-binaries.sh
           shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
-          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
-          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
-          echo '```' | tee -a ../tmp/checksums-formatted.txt
+
+      - name: Run Package build
+        run: |
+          constraintVersion="${{ github.ref_name }}"
+          ./cli/kctrl-linux-amd64 pkg release -y -v ${constraintVersion:1} --debug
+          mv ./carvel-artifacts/packages/kapp-controller.carvel.dev/metadata.yml ./carvel-artifacts/packages/kapp-controller.carvel.dev/package-metadata.yml
+          shasum -a 256 ./carvel-artifacts/packages/kapp-controller.carvel.dev/package.yml | tee -a ./tmp/checksums.txt
+          shasum -a 256 ./carvel-artifacts/packages/kapp-controller.carvel.dev/package-metadata.yml | tee -a ./tmp/checksums.txt
+          echo "# :open_file_folder: Files Checksum" | tee ./tmp/checksums-formatted.txt
+          echo '```' | tee -a ./tmp/checksums-formatted.txt
+          cat ./tmp/checksums.txt | tee -a ./tmp/checksums-formatted.txt
+          echo '```' | tee -a ./tmp/checksums-formatted.txt
```

Comment:
Any specific reason we have moved these commands in this step and removed it from the previous one?

Reply:
Moved this as we want to generate all the `shasums` first and then take care of coping checksums.txt into checksums-formatted.txt

Reply:
and if I directly generate shasum in checksums-formatted.txt the verify artifact step get fail as it verifies from checksums.txt

Reply:
As we can append to the file, I think it makes sense to move these commands up as those checksums are generated there. If you still think that formatted checksum need to be at one place, I would say create another step in Github Action something like `Add to formatted checksum`.

Reply:
Done